### DISCUSSION TITLE
Add example for $watch'ing $store to docs.

### DISCUSSION THREAD

**Discussion by james0r** [alpinejs/alpine#1546] [3. Feature ideas]
I see this is possible just like in Spruce, so it'd probably be of interest to others.

Did a demo to confirm it works -> https://codepen.io/James0r/pen/vYxPeqq?editors=1111

**Comment by zepfietje** [comment]
I think it's a good idea to add this to the docs indeed. Ended up here while trying to figure out how it works. Thanks @james0r!

```html
<div x-init="$watch('$store.darkMode.on', value => console.log(value))"></div>
```

**Reply by ju5t** [reply]
This was doing my head in. Thanks for the solution @james0r and @zepfietje!
